lang/values.md

```diff
@@ -80,7 +80,7 @@ impl Type {
         if bytes.len() != 1 {
             throw!();
         }
-        match bytes[0] {
+        match bytes[BigInt::zero()] {
```

Comment:
or do we want `impl Into<BigInt>`?

Reply:
For indexing? I mean if it's not too much to ask... yeah indexing with any `impl Into<BigInt>` would be nice. :)